frontend/src/domains/DomainCard.js

```diff
@@ -119,6 +118,27 @@ export function DomainCard({ id, url, status, hasDMARCReport, tags, ...rest }) {
             <Trans>Domain:</Trans>
           </Text>
           <Text isTruncated>{url}</Text>
+          <ABTestingWrapper insiderVariantName="B">
+            <ABTestVariant name="B">
+              <Flex>
```

Comment:
Consider giving this a `flex-wrap: wrap` in case multiple tags used on smaller screen